setup.py

```diff
@@ -1,56 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I would have expected this to be a rename (e.g. to setup_peas.py), so that it becomes easier to see the merge of this file's contents in POCS.

Reply:
The `setup.py` works via `python setup.py [install|develop]` and is a standard python name. The one file will install both `peas` and `pocs` when run based on how it is set up.